Source/Web/app/css/fullScreen.css

```diff
@@ -0,0 +1,13 @@
+/* This css will stop users from using their mobile phones to use this website*/
+/* This is largely because statistical models will not display properly*/
+@media only screen and (max-width: 991px) {
```

Comment:
This is an extremely large size to restrict to. Most of the website is still functional at this size. Consider just not showing the summary panel and just the login panel when it can't fit anymore.

Reply:
I agree on this one actually.